src/App.tsx

```diff
@@ -1,6 +1,14 @@
+import Home from "./Components/Home";
```

Comment:
we should probably consider a name other than home for the part where you write text